scripts/veff/enformer/enformer_vcf.smk

```diff
@@ -0,0 +1,77 @@
+import os
+
+SNAKEFILE = workflow.included_stack[-1]
+SNAKEFILE_DIR = os.path.dirname(SNAKEFILE)
+SCRIPT = os.path.basename(SNAKEFILE)[:-4]
+
+OUTPUT_BASEDIR = f"{VEFF_BASEDIR}/{SCRIPT}"
+RAW_VCF_PQ_PATTERN = f"{OUTPUT_BASEDIR}/raw.parquet/{{vcf_file}}.parquet"
+AGG_VCF_PQ_PATTERN = f"{OUTPUT_BASEDIR}/agg.parquet/{{vcf_file}}.parquet"
+TISSUE_VCF_PQ_PATTERN = f"{OUTPUT_BASEDIR}/tissue.parquet/{{vcf_file}}.parquet"
+VEFF_VCF_PQ_PATTERN = f"{OUTPUT_BASEDIR}/veff.parquet/{{vcf_file}}.parquet"
+
+rule enformer_predict_alternative:
+    resources:
+        mem_mb=lambda wildcards, attempt, threads: 12000 + (1000 * attempt),
+        gpu=1,
+    output:
+        temp(RAW_VCF_PQ_PATTERN)
+    input:
+        gtf_path=rules.enformer_gtf_to_parquet.output[0],
+        fasta_path=FASTA_FILE,
+        vcf_path=STRIPPED_VCF_FILE_PATTERN
+    params:
+        type='alternative'
+    conda:
+        # requires GPU on environment installation
```

Comment:
why does it require a GPU on installation?

Reply:
I forgot that erase this comment. We only need a GPU if we want to install tensorflow with GPU support (i.e. use the GPU conda environment).